### PR TITLE
Bug 1138350 - [Messages][RTL] Ellipsis direction fix in thread list view, r=azasypkin

### PR DESCRIPTION
**Pull request by steveck-chung**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[steveck-chung steveck-chung](https://github.com/steveck-chung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7a1fcc008e89)

File: apps/sms/style/sms.css [hunk 1]
```diff
@@ -232,6 +231,15 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   pointer-events: none;
 }
 
+[data-type="list"] .summary {
```
Comment (azasypkin) [inline apps/sms/style/sms.css]: Just curious why not smth like this ```.threadlist-item .summary``` and ```.threadlist-item .summary time```?

Comment (azasypkin) [inline apps/sms/style/sms.css]: Also I see that long LTR name in RTL or long RTL name in LTR is too close to the contact image [1]. Could you please fix this as well, while we're here?

[1] https://people.mozilla.org/~azasypkin/bug-1138350/issue.png

**Comment by try-server-hook** [comment]
[steveck-chung steveck-chung](https://github.com/steveck-chung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=61aea8a6764a)

**Comment by try-server-hook** [comment]
[steveck-chung steveck-chung](https://github.com/steveck-chung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d182a3fe6eaa)

File: apps/sms/style/sms.css (hunk 1 above)
Comment (steveck-chung) [inline apps/sms/style/sms.css]: Since .summary only exists in thread list, having addtional ```.threadlist-item``` seems not that necessary?

Comment (azasypkin) [inline apps/sms/style/sms.css]: Yeah, I'm okay with single selector as well, I just meant that don't really like '[data-type="list"] part, it looks too generic while it's for thread list only. Maybe it's better to rename to smth like ".threadlist-item-summary" as oocss evangelizes. But up to you :)

Comment (julienw) [inline apps/sms/style/sms.css]: I agree with @azasypkin here but let's do this in a separate bug as this one is 2.2+ and we should change all similar rules in one go.

**Comment by try-server-hook** [comment]
[steveck-chung steveck-chung](https://github.com/steveck-chung) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=aa334a10eb9f)

**Pull request merged**